handbook/engineering/languages/bash.md

```diff
@@ -0,0 +1,45 @@
+# Bash style
+
+Bash is frequently used in our build, CI and deployment systems. Some general guidelines and recommended reading
+
+- [Bash style](#bash-style)
+  - [Reading from a file](#reading-from-a-file)
+  - [Set -eu -o pipefail](#set--eu--o-pipefail)
+  - [References](#references)
+
+### Reading from a file
+
+Prefer using `mapfile` instead of `while IFS` to read a file
+
+```bash
+mapfile -t myArray < file.txt
```

Comment:
You can also do `mapfile -t myArray < <(find -d .)` to do it with the output of a command.

Reply:
Keep the tips coming! 😄